src/components/fields/Rating/Icon.tsx

```diff
@@ -0,0 +1,28 @@
+import RatingIcon from "@mui/icons-material/Star";
+import RatingOutlineIcon from "@mui/icons-material/StarBorder"
+import { get } from "lodash-es";
+
+
+export interface IIconProps{
+  config: any,
+  isEmpty: boolean
+}
+
+export default function ({config, isEmpty} : IIconProps) {
```

Comment:
```suggestion
export default function Icon({config, isEmpty} : IIconProps) {
```
Please name function components, so they appear correctly in React DevTools.